CSV_Modifier_Client/Services/AwsSecretsService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Amazon;
+using Amazon.SecretsManager;
+using Amazon.SecretsManager.Model;
+
+namespace CSV_Modifier_Client.Services
+{
+    public class AwsSecretsService
+    {
+        public async Task<string> GetAwsSecret (string secretName)
+        {
+            const string region = "us-east-1";
+            var awsSecretsMngrClient = new AmazonSecretsManagerClient(RegionEndpoint.GetBySystemName(region));
+
+            var request = new GetSecretValueRequest
+            {
+                SecretId = secretName,
+                VersionStage = "AWSCURRENT",
```

Comment:
This should be somewhere in appSettings.json

Reply:
done